prow/plugins/bugzilla/bugzilla.go

```diff
@@ -320,6 +340,55 @@ func (e *event) comment(gc githubClient) func(body string) error {
 	}
 }
 
+type queryUser struct {
+	Login githubql.String
+}
+
+type queryNode struct {
+	User queryUser `graphql:"... on User"`
+}
+
+type queryEdge struct {
+	Node queryNode
+}
+
+type querySearch struct {
+	Edges []queryEdge
+}
+
+/* emailToLoginQuery is a graphql query struct that should result in this graphql query:
+   {
+     search(type: USER, query: "email", first: 5) {
+       edges {
+         node {
+           ... on User {
+             login
+           }
+         }
+       }
+     }
+   }
+*/
+type emailToLoginQuery struct {
+	Search querySearch `graphql:"search(type:USER query:$email first:5)"`
+}
+
+// processQueryResult generates a response based on a populated emailToLoginQuery
+func processQuery(query *emailToLoginQuery, email string, log *logrus.Entry) string {
+	switch len(query.Search.Edges) {
+	case 0:
+		return fmt.Sprintf("ERROR: No GitHub users found with public email == %s", email)
+	case 1:
+		return fmt.Sprintf("/assign @%s", query.Search.Edges[0].Node.User.Login)
+	default:
+		response := fmt.Sprintf("ERROR: Multiple GitHub users found with public email == %s:", email)
```

Comment:
Similarly, formatting this with more prose would be better. Also let's make sure we're not at-mentioning people in this comment